### PR TITLE
Add info about needing to accept self-signed cert, fix test user

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,16 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+- Make sure that the binary was built with the above steps.
+- Now start all services with the following command
```
Comment (ownclouders) [inline README.md:46]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 2]
```diff
@@ -42,13 +42,16 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+- Make sure that the binary was built with the above steps.
```
Comment (ownclouders) [inline README.md:45]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 3]
```diff
@@ -42,13 +42,16 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+- Now start all services with the following command
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+- Open [https://localhost:9200](https://localhost:9200)
+- Accept the self-signed certificate (it is regenerated every time the server starts)
```
Comment (ownclouders) [inline README.md:53]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 4]
```diff
@@ -42,13 +42,16 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+- Make sure that the binary was built with the above steps.
+- Now start all services with the following command
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+- Open [https://localhost:9200](https://localhost:9200)
```
Comment (ownclouders) [inline README.md:52]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 5]
```diff
@@ -42,13 +42,16 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+- Accept the self-signed certificate (it is regenerated every time the server starts)
+- Login using one of the demo accounts:
```
Comment (ownclouders) [inline README.md:54]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 6]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+- Make sure that the binary was built with the above steps.
+- Now start all services with the following command
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+- Open [https://localhost:9200](https://localhost:9200)
+- Accept the self-signed certificate (it is regenerated every time the server starts)
```
Comment (ownclouders) [inline README.md:53]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 7]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+- Login using one of the demo accounts:
```
Comment (ownclouders) [inline README.md:54]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 8]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+- Make sure that the binary was built with the above steps.
```
Comment (ownclouders) [inline README.md:45]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 9]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+- Now start all services with the following command
```
Comment (ownclouders) [inline README.md:46]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 10]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+- Open [https://localhost:9200](https://localhost:9200)
```
Comment (ownclouders) [inline README.md:52]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 11]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+  - Make sure that the binary was built with the above steps.
+  - Now start all services with the following command
```
Comment (ownclouders) [inline README.md:46]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 12]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+  - Make sure that the binary was built with the above steps.
```
Comment (ownclouders) [inline README.md:45]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 13]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+  - Now start all services with the following command
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+  - Open [https://localhost:9200](https://localhost:9200)
+  - Accept the self-signed certificate (it is regenerated every time the server starts)
```
Comment (ownclouders) [inline README.md:53]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 14]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+  - Make sure that the binary was built with the above steps.
+  - Now start all services with the following command
 
 ```console
 ./bin/ocis server
 ```
 
-Open [https://localhost:9200](https://localhost:9200) and login using one of the demo accounts:
+  - Open [https://localhost:9200](https://localhost:9200)
```
Comment (ownclouders) [inline README.md:52]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md [hunk 15]
```diff
@@ -42,16 +42,19 @@ mkdir -p /var/tmp/reva/root/{home,oc}
...
+  - Accept the self-signed certificate (it is regenerated every time the server starts)
+  - Login using one of the demo accounts:
```
Comment (ownclouders) [inline README.md:54]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

Comment (PVince81) [reply README.md:54]: wtf, you just told me to add 2 spaces...

File: README.md (hunk 6 above)
Comment (ownclouders) [inline README.md:53]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md (hunk 9 above)
Comment (ownclouders) [inline README.md:46]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md (hunk 10 above)
Comment (ownclouders) [inline README.md:52]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md (hunk 7 above)
Comment (ownclouders) [inline README.md:54]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md (hunk 8 above)
Comment (ownclouders) [inline README.md:45]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5426122)

File: README.md (hunk 6 above)
Comment (PVince81) [reply README.md:53]: look, and now it's asking me to add 2 spaces again

**Comment by PVince81** [comment]
previous attempt: adding white spaces between the lines and indenting the code blocks
next attempt: adding 2 spaces not before the dash but **after** the dash...

**Comment by PVince81** [comment]
image

so apparently this is what it wanted, but the error message was not helpful... why so many spaces ?!

anyway, I bowed to this bot's will now after losing my sanity

squashing...

**Comment by micbar** [comment]
@PVince81 please rebase

**Comment by PVince81** [comment]
rebased

**Pull request merged**